pyramid_oereb/lib/records/documents.py

```diff
@@ -1,7 +1,10 @@
 # -*- coding: utf-8 -*-
+import logging
 import warnings
 import datetime
 
+LOG = logging.getLogger(__name__)
```

Comment:
Nitpicking: can we keep the log variable lowercase?

Reply:
OK

Reply:
thanks